From: Jochen Sprickerhof <git@jochen.sprickerhof.de>
Date: Mon, 8 Apr 2019 21:06:57 +0200
Subject: Don't test buffers with char

As we have tests with negative values, char is not a fitting data type
as it resulted in broken tests on arm.
---
 test/io/test_buffers.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/io/test_buffers.cpp b/test/io/test_buffers.cpp
index ab254ae..da3e4c5 100644
--- a/test/io/test_buffers.cpp
+++ b/test/io/test_buffers.cpp
@@ -81,7 +81,7 @@ class BuffersTest : public ::testing::Test
 
 };
 
-typedef ::testing::Types<char, int, float> DataTypes;
+typedef ::testing::Types<int, float> DataTypes;
 TYPED_TEST_CASE (BuffersTest, DataTypes);
 
 TYPED_TEST (BuffersTest, SingleBuffer)
